app/__init__.py

```diff
@@ -5,7 +5,18 @@
 load_dotenv()
 app = Flask(__name__)
 
+about_us_text = [
```

Comment:
I like this solution to hardcoding HTML. For consistency with rest of project, we could also store this data in a JSON file in /data directory, but I do not think that is neccessary. What do other team members think?

Reply:
Just did it!